src/renderers/webgl/WebGLPrograms.js

```diff
@@ -259,7 +259,8 @@ function WebGLPrograms( renderer, cubemaps, cubeuvmaps, extensions, capabilities
 			doubleSided: material.side === DoubleSide,
 			flipSided: material.side === BackSide,
 
-			depthPacking: ( material.depthPacking !== undefined ) ? material.depthPacking : false,
+			useDepthPacking: !! material.depthPacking,
+			depthPacking: material.depthPacking || 0,
```

Comment:
I thought using two variables makes the code more clear. `useDepthPacking` defines whether depth packing should be used or not. `depthPacking` defines the type of depth packing.